### PR TITLE
bytefield:0.0.1

### PR DESCRIPTION
**Pull request by jomaway**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: Simple way to create protocol header diagrams in typst.

### CODE REVIEW

File: packages/preview/bytefield/0.0.1/typst.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[package]
+name="bytefield"
+version = "0.0.1"
+entrypoint = "bytefield.typ"
+authors = ["Jomaway <https://github.com/jomaway>"]
+license = "MIT"
```
Comment (laurmaedje) [inline packages/preview/bytefield/0.0.1/typst.toml:6]: could you `exclude` the `ipv4-example.png` file to reduce the package size? there are a bunch of other packages which exclude assets, you can take a look there.

File: packages/preview/bytefield/0.0.1/typst.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[package]
+name="bytefield"
```
Comment (laurmaedje) [inline packages/preview/bytefield/0.0.1/typst.toml:2]:
```suggestion
name = "bytefield"
```

File: packages/preview/bytefield/0.0.1/typst.toml (hunk 1 above)
Comment (jomaway) [reply packages/preview/bytefield/0.0.1/typst.toml:6]: Sure, I will do after my vacation.

Comment (jomaway) [reply packages/preview/bytefield/0.0.1/typst.toml:6]: @laurmaedje removed the `png` as requested.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**